### PR TITLE
Add videosink example

### PR DESCRIPTION
**Pull request by hansmi**
As discussed in periph/devices#26. The example command uses only periph.io and the standard library.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,6 +9,6 @@ go 1.13
 require (
 	periph.io/x/conn/v3 v3.6.9
 	periph.io/x/d2xx v0.0.4
-	periph.io/x/devices/v3 v3.6.12
+	periph.io/x/devices/v3 v3.6.13-0.20211210231915-5b35a3fb05bf
```
Comment (hansmi) [inline go.mod]: This is necessary because no tagged version contains `videosink` yet.

@maruel, could you perhaps tag a release?

Comment (maruel) [reply go.mod]: Will do.

Comment (maruel) [reply go.mod]: Done, please rebase.

Comment (maruel) [reply go.mod]: I resolved the conflict with a merge in the meantime.

Comment (maruel) [reply go.mod]: It worked, so I rebased and squashed it.

**Pull request merged**